src/compiler/parse.jl

```diff
@@ -152,3 +167,18 @@ function parse!(
 
     return nothing
 end
+
+function parse!(
+    model::Virtual.Model{T},
+    g::PBO.PBF{VI,T},
+    f::SQF{T},
+    s::GT{T},
+    arch::AbstractArchitecture,
+) where {T}
+    parse!(model, g, f, arch)
+
+    g[nothing] -= s.lower
+
+    return nothing
+end
+
```

Comment:
There are several warnings from codecov where lines are not covered by tests. Should we work on these before merging?

Reply:
Most are due to new attributes, it will be straightforward to cover them.